Common/Sources/View/Error/ErrorView.swift

```diff
@@ -57,7 +57,11 @@ public final class ErrorView {
             connectionError({ (_) in handler?() })
 
         default:
-            unexpectedError()
+            if error.getAuthenticationError() != nil {
```

Comment:
FYI: In android we worked like this.
<img width="761" alt="Screenshot 2020-03-19 at 11 46 44 AM" src="https://user-images.githubusercontent.com/52773997/77037458-687b0800-69d7-11ea-8ac6-eaf8fe438877.png">
<img width="929" alt="Screenshot 2020-03-19 at 11 47 09 AM" src="https://user-images.githubusercontent.com/52773997/77037466-6d3fbc00-69d7-11ea-9a92-dec54d173e05.png">

Reply:
But in ios that was the requirement

Reply:
Good point Mani. We can actually do similar thing in iOS. @asenthil-altmtrk instead of checking for error.getAuthenticationError() != nil in UI SDK and iOS App, update `public var group: HyperwalletErrorGroup ` in HyperwalletError.swift (Hyperwallet Core SDK) to add another case for authenticationError from 
```suggestion
            case .authenticationError:
                return HyperwalletErrorGroup.authenticationError
```
and remove it from unexpected case.

Reply:
Also then update ErrorView in HyperwalletApp `case .unexpected:
            if error.getAuthenticationError() != nil {
                authenticationError(title, handler)
            } else {
                unexpectedError({ (_) in handler?() })
            }
        }` to check for .authenticationError instead.

Reply:
I have updated in Core SDK, UISDK  and App